components/dialog/stories/template.js

```diff
@@ -94,13 +115,29 @@ export const Template = ({
 
 	if (showModal) {
 		return html`
+		${Button({
+				size: "m",
+				variant: "secondary",
+				label: "Click to open dialog",
+				treatment: "outline",
+				customClasses: [],
+				customStyles: {
+					position: "absolute",
+					insetInlineStart: "50%",
+					insetBlockStart: "50%",
+					transform: "translate(-50%, -50%)",
+				},
+				onclick: toggleOpen,
+			}, context)}
```

Comment:
```suggestion
```

I'm not sure we need this in this template - the `isOpen` boolean will be able to toggle the hide/show of the Dialog without adding a button to the Storybook UI.

Reply:
Reverted ✔️